released/packages/coq-io-hello-world/coq-io-hello-world.1.2.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "dev@clarus.me"
+homepage: "https://github.com/clarus/coq-hello-world"
+dev-repo: "git+https://github.com/clarus/coq-hello-world.git"
+bug-reports: "https://github.com/clarus/coq-hello-world/issues"
+authors: ["Guillaume Claret"]
+license: "MIT"
+build: [
+  ["./configure.sh"]
+  [make "-j%{jobs}%"]
+  [make "-C" "extraction"]
+]
+install: [
+  ["install" "-T" "extraction/main.native" "%{bin}%/helloWorld"]
+]
+depends: [
+  "coq-io" {>= "4.0.0" build}
+  "coq-io-system" {build}
+  "coq-io-system-ocaml" {>= "2.3.0"}
+]
+tags: [
+  "keyword:effects"
+  "keyword:extraction"
```

Comment:
Please add a `"date:YYYY-MM-DD"` tag (indicating when you did the release on GitHub).